src/components/Timelines.js

```diff
@@ -46,12 +46,20 @@ function Timelines({dataPath=`/api/v1/timelines`, include_repo = false, gh_url="
     }, [loading]);
 
     function fetchTimelines() {
-        fetch(`${dataPath}?offset=${offset}&limit=${PER_CALL}`).then(res => res.json()).then(data => {
+        fetch(`${dataPath}?offset=${offset}&limit=${PER_CALL}`).then(res => {
+            if (!res.ok) {
+                notifications.sendError(`Failed to fetch data.`, `${res.status}: ${res.statusText}`);
+                return undefined;
```

Comment:
I think you can just `return` here, and then the value returned would be `undefined`.